### ISSUE TITLE
Plans for this library

### ISSUE THREAD

**Issue by SamMousa** [goaop/parser-reflection#139]
Are you planning to maintain this package / bring it back to life?
Its readme shows a deprecation message which caused us to deprecate `Codeception/AspectMock`: https://github.com/Codeception/AspectMock/issues/208

What are your plans moving forward? Is goaop itself moving to `BetterReflection`?

**Comment by aik099** [comment]
I'm also interested.

I'm using this library In the https://github.com/console-helpers/code-insight project. The `ReflectionFile` class concept of this library is a killer feature because it allows me to detect all supported reflectable stuff in a PHP file. No analog in the `BetterReflection`.

**Comment by lisachenko** [comment]
Yep, ReflectionFile, ReflectionFilenamespace is what makes my library
easier for use. If someone could port this to BR - I would be very happy to
switch to the BetterReflection, as it has maintainer from Psalm…

Пн, 16 дек. 2024 г. в 22:32, Alex ***@***.***>:

**Comment by aik099** [comment]
They won't accept this. I've asked about this in the https://github.com/Roave/BetterReflection/issues/732 already.

I've tried to extract `ReflectionFile` and `ReflectionFilenamespace` classes previously to be used with a `BetterReflection`, but due to absolutely different AST Tree building concepts (when to build it and who can scan it) I had almost no success with it.

**Comment by lisachenko** [comment]
This project will continue to receive support, now with LLM it is much more easier to do. Thanks!
